python_on_whales/components/service/cli_wrapper.py

```diff
@@ -173,6 +173,10 @@ def create(
         limit_memory: Optional[str] = None,
         limit_pids: Optional[int] = None,
         log_driver: Optional[str] = None,
+        network: Optional[str] = None,
+        restart_condition: Optional[str] = None,
```

Comment:
I'm thinking about the type hint for restart_condition. It could be `Optional[Literal["none", "on-failure", "any"]]` no? What do you think?